hack/render-manifests.go

```diff
@@ -45,14 +47,23 @@ func main() {
 		exitWithError(err, "failed to create output dir %s", outputDir)
 	}
 
-	tmpl, err := template.ParseGlob(path.Join(inputDir, "*.yaml"))
+	var tmpl *template.Template
+	tmpl, err = template.ParseGlob(path.Join(inputDir, "operator/*.yaml"))
 	if err != nil {
-		exitWithError(err, "failed parsing top dir manifests at %s", inputDir)
+		exitWithError(err, "failed parsing top dir operator manifests at %s", inputDir)
 	}
 
-	tmpl, err = tmpl.ParseGlob(path.Join(inputDir, "*/*.yaml"))
+	// Be explicit about which subdirs we render. Otherwise, we might inadvertently override
+	// a manifest with the same name. Once the operator is in place, we shouldn't need
+	// to do this anymore.
+	tmpl, err = tmpl.ParseGlob(path.Join(inputDir, "crds/*.yaml"))
```

Comment:
Don't we have to render just operator's crds ?

Reply:
Not sure we have to render CRDs at all, tbh. Are we doing this just to copy them? And we should probably add the handler namespace, pull policy, and image to the spec.

Reply:
You right, CRDs are generated let's just remove this line from here and copy them from deploy/crds directly into the containers.